test/sasdataloader/test/utest_sesans.py

```diff
@@ -0,0 +1,100 @@
+"""
+    Unit tests for the SESANS .ses reader
+"""
+
+import unittest
+from sas.sascalc.dataloader.readers.sesans_reader import Reader
+
+
+class sesans_reader(unittest.TestCase):
+
+    def setUp(self):
+        reader = Reader()
+        self.loader = reader.read
+
```

Comment:
I went with Reader because Loader swallows exceptions, making them impossible to test for.  I did switch one of the tests, which doesn't throw any exceptions, back onto the loader to confirm that the file format is properly detected.